src/helpers/steg/barnetsBostedEndre.ts

```diff
@@ -102,6 +117,7 @@ export const skalAnnenForelderRedigeres = (
   finnesBarnSomSkalHaBarnepassOgRegistrertAnnenForelderBlantValgteBarn: boolean
 ) => {
   return (
+    manueltUtfyltForelder(barn, barnHarSammeForelder) ||
```

Comment:
`barnHarSammeForelder: boolean | undefined,` 

Dette variabelnavnet var ikke trivielt for hjernen min i dag -mener vi her at barnets forelder er "kopiert" fra en annen?

Klarer vi å utlede det på en måte som gjør det forståelig for Gunnstein 2.0 som kommer inn i teamet om 3 år? :D